app/src/main/java/com/dodo/flashcards/domain/models/User.kt

```diff
@@ -0,0 +1,6 @@
+package com.dodo.flashcards.domain.models
+
+interface User {
+    val email: String
+    val username: String
+}
```

Comment:
The motivation for this was so that we don't expose `FirebaseAuth` or `FirebaseUser` to our ViewModels.